package.json

```diff
@@ -2,14 +2,13 @@
   "name": "meriyah",
   "version": "5.0.0",
   "description": "A 100% compliant, self-hosted javascript parser with high focus on both performance and stability",
-  "main": "dist/meriyah.cjs",
-  "module": "dist/meriyah.esm.js",
-  "jsnext:main": "dist/meriyah.esm.js",
+  "main": "dist/meriyah.cjs.js",
+  "module": "dist/meriyah.esm.mjs",
```

Comment:
I think this `module` entry should link to minified version.

Import from  `https://unpkg.com/meriyah?module` will use it, jsdelivr should also use it I guess.

I think only CDNs and bundlers will look for this field.

Reply:
Done.